Library/CheckoutModels.swift

```diff
@@ -12,7 +12,7 @@ public struct PaymentTokenData {
 
 public struct PaymentMethodData {
   public let displayName: String?
-  public let network: String?
+  public let network: PKPaymentNetwork?
```

Comment:
`PKPaymentNetwork` is now a proper enum so we can handle it directly instead of through it's `rawValue` interface.